libiocage/cli/start.py

```diff
@@ -49,8 +49,11 @@ def cli(ctx, rc, jails, log_level):
     for jail in ioc_jails.list(filters=jails):
         logger.log(f"Starting {jail.humanreadable_name}")
         try:
-            jail.start()
+            for i in jail.start(yields=True):
+                print(f"[+] {i.action} OK")
+
         except Exception:
+            raise
             exit(1)
```

Comment:
I'm sorry, but this is necessary to see python stacktraces, so I have this enabled until I prepare the branch for the merge. The errors that get swallowed from exit(1) are structural ones, not feature failures, so they are useful for development only. Already told @skarekrow that I'll find a better solution for this, because it lead to quite some confusion. For example this one https://github.com/iocage/libiocage/pull/35#discussion_r135416094

:trollface: